pkg/console/operator/sync_v400.go

```diff
@@ -593,3 +602,15 @@ func (co *consoleOperator) GetAvailablePlugins(enabledPluginsNames []string) []*
 	}
 	return availablePlugins
 }
+
+func getNodeArchitectures(nodes *corev1.NodeList) []string {
+	nodeArchitectures := []string{}
+	for _, node := range nodes.Items {
+		nodeArch := node.Labels[api.NodeArchitectureLabel]
+		if !util.Contains(nodeArchitectures, nodeArch) {
+			nodeArchitectures = append(nodeArchitectures, nodeArch)
+		}
+	}
+	sort.Strings(nodeArchitectures)
+	return nodeArchitectures
```

Comment:
Nit: It might be slightly more efficient to use [`sets.String`](https://pkg.go.dev/k8s.io/apimachinery/pkg/util/sets#String) here.

```suggestion
	nodeArchitectures := sets.NewString()
	for _, node := range nodes.Items {
		nodeArch := node.Labels[api.NodeArchitectureLabel]
		nodeArchitectures.Insert(nodeArch)
	}
	return nodeArchitectures.List()
```